### ISSUE TITLE
Database in config required

### ISSUE THREAD

**Issue by jlloyd-widen** [MeltanoLabs/tap-mysql#10]
Running the tap for the first time I got this error:

```
File "/opt/dagster/app/.meltano/extractors/tap-mysql--collective/venv/lib/python3.8/site-packages/tap_mysql/tap.py", line 170, in get_sqlalchemy_url
    database=config["database"],

KeyError: 'database'
```

The docs indicate that this is not a required parameter, indeed, it would be a regression compared to previous versions of this tap.

**Comment by visch** [comment]
Yeah we can definitely remove that. So you know right now the database doesn't even limit you to just that database anyway so it's pretty pointless right now. Maybe we should use the database parameters instead of filter db option
